python/ray/dashboard/client/src/pages/metrics/Metrics.tsx

```diff
@@ -96,15 +96,19 @@ const METRICS_CONFIG: MetricsSectionConfig[] = [
         pathParams: "orgId=1&theme=light&panelId=26",
       },
```

Comment:
BTW, once we add the "rate of finished tasks / second", We should be able to get rid of this "Scheduler Task State" graph. This graph is not very performant because of the `max_over_time` function.

Reply:
I'll create another PR for the rate graph after addressing the null initilization issue